partiql-spi/src/test/kotlin/org/partiql/spi/connector/sql/SqlHeader.kt

```diff
@@ -1,831 +0,0 @@
-package org.partiql.spi.connector.sql
```

Comment:
I'm assuming all of the headers will be generated as part of https://github.com/partiql/partiql-lang-kotlin/pull/1490?

Reply:
Yes